index.html

```diff
@@ -1,14 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
     <link rel="stylesheet" href="./styles/styles.css">
     <title>Larocca Neumáticos</title>
 </head>
+
 <body>
-    <h1 class="mainTittle">Larocca Neumáticos (proyecto de javaScript)</h1>
+    <div id="logoPrincipal">
+        <img src="./assets/logoLarocca.png" alt="logoPrincipal">
+    </div>
+    <header>
+        <div class="mainMenu">
+            <ul class="">
+                <li>Home</li>
+                <li>Neumaticos</li>
+                <li>Accesorios</li>
+                <li>Servicios</li>
+                <li>Nosotros</li>
+                <div class="wrapCarrito">
+                    <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#carritoCompras">
+                        <i class="bi bi-cart4"><span id="contadorCarrito"> (0)</span></i>
```

Comment:
No cerraste el elemento `<button>`.
```suggestion
                    <button
                      type="button"
                      class="btn btn-primary"
                      data-bs-toggle="modal"
                      data-bs-target="#carritoCompras"
                    >
                        <i class="bi bi-cart4"><span id="contadorCarrito"> (0)</span></i>
                    </button>
```